ansible_collections/arista/avd/roles/eos_cli_config_gen/README_v4.0.md

```diff
@@ -669,7 +669,8 @@ ip_extcommunity_lists:
 
 ```yaml
 ip_extcommunity_lists_regexp:
-  < community_list_name >:
+  - name: < community_list_name >
+    types:
     - type: < permit | deny >
```

Comment:
missing two extra spaces to be consistent with other models. This is also missing from the molecule file - causing the linter error in CI